lib/dpkg.sh

```diff
@@ -132,7 +132,18 @@ dpkg_Sy() {
 }
 
 dpkg_Ss() {
-  apt-cache search "$@"
+  IFS=$'\n' packages=($(apt-cache search "$1"))
```

Comment:
Wrap this in a subshell, otherwise, `IFS` is changed permanently.

Reply:
IFS is for array, it's applied to packages variable. If you change to subshell, the packages will use space, then each word became a member of array

Reply:
`dpkg_Ss` can be invoked without any option (when "$1" is not defined). The first expression may generate an error in that situation. Maybe we need `apt-cache search "${@}"` instead?

Reply:
@NgoHuy What I mean is wrapping the whole function in a subshell, otherwise, your change to `IFS` is permanent and affect other functions.

Reply:
Or in short, define your function as:

```
dpkg_Ss() (
  : rest of your body here
)
```

Reply:
I dont think it will affected other function, when dkpg_Ss is calling, it change once, and when other is calling, it does not touch the IFS

Reply:
yes it does @NgoHuy ,, that's how Bash is crazy =))

Reply:
I push commit to fix with local variable of function level.

Reply:
@icy I also changed $1 to $@ as accepted many keywords